src/3box.js

```diff
@@ -93,7 +93,7 @@ class Box {
 
     const onNewPeer = async (topic, peer) => {
       if (peer === this.pinningNode.split('/').pop()) {
-        this._pubsub.publish(PINNING_ROOM, { type: 'PIN_DB', odbAddress: rootStoreAddress })
+        this._pubsub.publish(PINNING_ROOM, { type: 'PIN_DB', odbAddress: rootStoreAddress, did: this._3id.getDid() })
```

Comment:
We do another `PIN_DB` in the `_ensurePinningNodeConnected` method, should we send the did from there too?

Reply:
I don't think that is really needed. When that is called we know that the DID will already have been sent to the pinning node once!